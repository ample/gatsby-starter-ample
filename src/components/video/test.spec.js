import React from "react"
import renderer from "react-test-renderer"

import { component as Video, fixtures } from "."

describe("Video", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Video {...fixtures.default} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
