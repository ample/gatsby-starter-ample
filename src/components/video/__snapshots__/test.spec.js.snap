// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video renders correctly 1`] = `
<div
  className="video"
>
  <iframe
    allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
    allowFullScreen={true}
    frameBorder="0"
    height="315"
    src="https://www.youtube.com/embed/cIcNqn89BoU"
    title="cIcNqn89BoU"
    width="560"
  />
</div>
`;
